proposals/dao/cr_fix.ts

```diff
@@ -0,0 +1,232 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { DeployUpgradeFunc, SetupUpgradeFunc, TeardownUpgradeFunc, ValidateUpgradeFunc } from '../../types/types';
+import { BPTLens, CollateralizationOracle, StaticPCVDepositWrapper } from '@custom-types/contracts';
+
+chai.use(CBN(ethers.BigNumber));
+
+const CHAINLINK_LUSD = '0x3D7aE7E594f2f2091Ad8798313450130d0Aba3a0';
+const CHAINLINK_BAL = '0xC1438AA3823A6Ba0C159CfA8D98dF5A994bA120b';
+const CHAINLINK_CREAM = '0x82597CFE6af8baad7c0d441AA82cbC3b51759607';
+
+/*
+
+CR Oracle updates
+
+Deploy:
+1. Lusd-usd oracle
+2. Bal-Eth oracle
+3. Cream-Eth oracle
+4. Bal-usd composite oracle
+5. Cream-usd composite oracle
+6. Fei LUSD LBP Lens
```

Comment:
why bother, the auction ends in 3 days ?

Reply:
hmm maybe it makes sense to keep it, if we do another auction in the future

Reply:
Good point. Its mostly to see what the oracle will return